index.html

```diff
@@ -841,9 +841,9 @@
               </li>
               <li>Determine which <a>payment handlers</a> support
               <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <var>data</var> contains filters that exclude the payment
```

Comment:
Looking at this text (including the one in the spec), it actually doesn't make any sense at all... It's a case of "take these identifiers and their corresponding data, and let god sort it out." That's all we can really say here. Right now it's super hand-wavy on purpose.